commons/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
     <uses-permission android:name="android.permission.READ_SYNC_STATS"/>
     <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
```

Comment:
Oh... that was actually me trying to fix logcat errors that had already existed before I made any changes to the code. The error was 

```
"Permission Denial: opening provider com.android.providers.media.MediaDocumentsProvider from ProcessRecord{2679a9d5 2524:fr.free.nrw.commons/u0a53} (pid=2524, uid=10053) requires android.permission.MANAGE_DOCUMENTS or android.permission.MANAGE_DOCUMENTS"
```

But it doesn't seem to have fixed the logcat errors, and my changes to the code don't need it. So I can revert that additional permission.